src/System.Private.Reflection.Metadata.Ecma335/src/Shims/RuntimeReflectionExtensions.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Reflection
+{
+    internal static partial class RuntimeReflectionExtensions
+    {
+        public static MethodInfo GetRuntimeMethod(this Type type, String name, Type[] parameters)
```

Comment:
Running reflection in the middle of type loader for light-up is a land mine that we will step on sooner or later. Should we disable the reflection light up code instead?